go.mod

```diff
@@ -5,8 +5,8 @@ go 1.20
 require (
 	github.com/hashicorp/golang-lru v0.6.0
 	github.com/multiversx/concurrent-map v0.1.4
-	github.com/multiversx/mx-chain-core-go v1.2.19-0.20231129100534-356aa234f4ff
-	github.com/multiversx/mx-chain-logger-go v1.0.14-0.20231129101244-c44fa1c79b03
+	github.com/multiversx/mx-chain-core-go v1.2.19-0.20240129082057-a76d0c995cf2
+	github.com/multiversx/mx-chain-logger-go v1.0.14-0.20240104072921-bf87e7d0a729
```

Comment:
maybe use the version from here? https://github.com/multiversx/mx-chain-logger-go/pull/26

Reply:
done